packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -51,6 +52,13 @@ import {
   partitionPopperPropsFromShorthand,
 } from '../../utils/positioner';
 
+export type unhandledA11yProps = {
```

Comment:
Please why do you call the type `unhandledA11yProps` here and `a11yProps` in `Datepicker`?
Is there any difference in how they are handled?